### ISSUE TITLE
H = tf.keras.layers.Conv2D(?????)(X) 뒤에 (X)의 문법이 궁금합니다.

### ISSUE THREAD

**Issue by miracle153f** [codingeverybody/codingyahac#1161]
# 궁금한 점
정말 이해하기 쉽게 강의해 주셔서 감사합니다.
공부하는 중 문법적인 부분에서 궁금한 점이 생겨서 질문드립니다.

X = tf.keras.layers.Input(shape=[28, 28, 1])

H = tf.keras.layers.Conv2D(6, kernel_size=5, padding='same', activation='swish')(X)

위 코드에서, 마지막 (X) 부분을 쓰는 이유는 알겠는데, 문법적으로는 잘 이해가 되지 않습니다.

보통 소괄호는 연산 순서를 바꾸거나, abs(x)처럼 함수의 매개변수를 쓰기 위한 것인데,

Conv2D()함수 뒤에 (X)가 나와서 어떤 문법적 내용인지 잘 모르겠습니다.

설명 주시면 감사하겠습니다.
